### ISSUE TITLE
Use asp-* attributes

### ISSUE THREAD

**Issue by zHaytam** [HtmlTags/htmltags#101]
Hello,

Is it possible for example to generate a html element (e.g. span) and use for exaxmple `asp-validation-for` on it?

If no, are you aware of any way to manually call these kind of methods? `asp-for`, `asp-action` etc...

Thank you!

**Comment by jbogard** [comment]
Yes, you can, though those are done through tag helpers, a different
extension. It's possible to combine some together, but you have to know how
ASP.NET Core's light up (is it just an attribute, or attribute and element).

**Comment by zHaytam** [comment]
What I want to do is write write as a string (or using HtmlTags) an input element for example and add `asp-for` attribute to it and I want the taghelper to execute but it doesn't because it doesn't get processed I end up with the attribute in the html like it's a normal attribute.

**Comment by jbogard** [comment]
Ah right because tag helpers are weird....are you using the HtmlTags tag helpers too? I assumed those would layer (basically it's 2 tag helpers executing).
